app/knight/knight_class.py

```diff
@@ -0,0 +1,51 @@
+class Knight:
+    protection = 0
+    list_of_knights = {}
+
+    def __init__(self, knight: object) -> None:
```

Comment:
In Python everything is an object, be more specific in type hinting throughout the project.

Reply:
+